pegasus/sites.v3/code.org/views/donor_slider.haml

```diff
@@ -21,3 +20,9 @@
             = "#{supporter[:name_s]}"
           - else
             != "#{supporter[:name_s]}, "
+        :javascript
+          // Shuffle donor-slider links.
+          var ul = document.querySelector('.donor-slider');
+          for (var i = ul.children.length; i >= 0; i--) {
+            ul.appendChild(ul.children[Math.random() * i | 0]);
```

Comment:
The bitwise-or truncates a floating-point, to generate a random index-integer from `0` to `i-1`.